storage/oasis/nodeapi/damask/node.go

```diff
@@ -63,6 +63,20 @@ func (c *ConsensusApiLite) StateToGenesis(ctx context.Context, height int64) (*g
 	return &rsp, nil
 }
 
+func (c *ConsensusApiLite) GetConsensusParameters(ctx context.Context, height int64) (*nodeapi.ConsensusParameters, error) {
+	/*
+		var rsp consensus.Parameters
+		if err := c.grpcConn.Invoke(ctx, "/oasis-core.Consensus/GetParameters", nil, &rsp); err != nil {
+			return nil, fmt.Errorf("GetParameters(damask): %w", err)
+		}
+	*/
```

Comment:
Can we remove this commented out block? I think the example in `eden/node.go` should suffice, but open to debate. Same applies for `cobalt/node.go`.